### PR TITLE
Refaelbenzvi24/sc 35/gather all UI components export to one file

### PR DESCRIPTION
**Pull request by Refaelbenzvi24**
gathered all ui components export to UI/index.tsx

### CODE REVIEW

File: src/components/ReactQuery/QueryHandler.tsx [hunk 1]
```diff
@@ -1,27 +1,51 @@
...
+const QueryHandler = <LoadingComponent extends keyof typeof loadingComponents = "spinner">({
+	children,
+	status,
+	loadingComponent,
+	loadingComponentProps,
+}: QueryHandlerProps<LoadingComponent> & typeof defaultProps) => {
+	const LoadingComponent = loadingComponents[loadingComponent]
+
+	return (
+		<>
```
Comment (Noyai274) [inline src/components/ReactQuery/QueryHandler.tsx:41]: how about using switch case here?

**Review by Noyai274 — COMMENTED** [review]
It is considered good practice to arrange your imports if the following way:
1. out-sourced
2. internal pieces of code

for example:
instead of:
import { Link, useNavigate } from "react-router-dom"
import { Button, Col, Row, TextField, theme } from "../components/UI"
import { authEndpoint } from "../services"
import { useFormik } from "formik"
import * as Yup from 'yup'
import TokenStorage from "../modules/TokenStorage"
import { useEffect } from "react"
import ApiError from "../modules/ApiError"
import useErrorsHandler from "../hooks/useErrorsHandler"

doing this:
import { Link, useNavigate } from "react-router-dom"
import { useEffect } from "react"
import { useFormik } from "formik"
import * as Yup from 'yup'

import { Button, Col, Row, TextField, theme } from "../components/UI"
import { authEndpoint } from "../services"
import TokenStorage from "../modules/TokenStorage"
import ApiError from "../modules/ApiError"
import useErrorsHandler from "../hooks/useErrorsHandler"

File: src/components/ReactQuery/QueryHandler.tsx (hunk 1 above)
Comment (Refaelbenzvi24) [reply src/components/ReactQuery/QueryHandler.tsx:41]: See this [blog post](https://blog.cleancoder.com/uncle-bob/2021/03/06/ifElseSwitch.html) talking about if statements vs switch case. :)

Comment (Refaelbenzvi24) [reply src/components/ReactQuery/QueryHandler.tsx:41]: I have read the comment about ordering the imports and found this as an opportunity to search about it on google, I have found [this article](https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m) which's talking about it, and created a [story](https://app.shortcut.com/anki/story/37/create-a-role-for-imports-order) in Clubhouse as well.

**Review by Noyai274 — DISMISSED** [review]

**Pull request merged**